### ISSUE TITLE
Queues not processing

### ISSUE THREAD

**Issue by tfunq** [piwik/plugin-QueuedTracking#215]
We have a queue with 16 workers. Two of them does not get processed.

```bash
./console queuedtracking:monitor
Queue is enabled
Request sets in the queue will be processed automatically after a tracking request
Up to 16 workers will be used
Processor will start once there are at least 50 request sets in the queue
3959833 (0+5+1+11+25+1982970+1976521+16+36+26+33+47+46+35+25+36) request sets left in queue. 4.83G used memory (4.84G peak). 0 workers active.
```
Trying to force process them gives an error:

```bash
./console queuedtracking:process --queue-id=5
Forcing queue ID: 5
Starting to process request sets, this can take a while
Uncaught exception in ./vendor/matomo/device-detector/ClientHints.php line 193:
array_combine(): Argument #1 ($keys) and argument #2 ($values) must have the same number of elements
```

**Comment by AltamashShaikh** [comment]
@tfunq We just released version 4.0.7 of the plugin to make the queue work even when we face such issues, can you update the plugin and try again ?

**Comment by AltamashShaikh** [comment]
Looks like duplicate of #194 which is fixed by release of version 4.0.7, so closing this issue for now
